Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -396,6 +396,8 @@ It can efficiently compress both junk data and encrypted Wii data.
     <string name="emulation_toggle_controls">Toggle Controls</string>
     <string name="emulation_toggle_all">Toggle All</string>
     <string name="emulation_control_scale">Adjust Scale</string>
+    <string name="emulation_control_opacity">Adjust Opacity</string>
+    <string name="emulation_control_Adjustments">Adjust Controls</string>
```

Comment:
This should be `emulation_control_adjustments` with a lowercase a.

Additionally, I'm unsure if it makes sense to call this Adjust Controls (as it might give users the impression that this is where you should go for things like moving the controls), but I can't think of a name that's much better... I suppose Adjust Parameters avoids that specific problem, but it's still a rather unclear name.

Reply:
I understand what you mean but considering there is an Edit Layout button above it; I think it can still work out even for new users.